### PR TITLE
generate and use new validation params and arguments for RT validation, and load into bigquery

### PR DESCRIPTION
**Pull request by atvaccaro**
# Overall Description

Step towards closing https://github.com/cal-itp/data-infra/issues/787 and essentially closes https://github.com/cal-itp/data-infra/issues/786. This PR changes the RT validation to use v0.0.4 of the RT validator wrapper which will write out files identified with the itp_id, url, and entity type. It also points the external tables at the new, proper output paths for each entity type.

## Checklist for all PRs

- [x] Run `pre-commit run --all-files` to make sure markdown/lint passes
- [x] Link this pull request to all issues that it will close using keywords (see GitHub docs about [Linking a pull request to an issue using a keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)). Also mention any issues that are partially addressed or are related.

## Airflow DAG changes checklist

- [x] Verify that all affected DAG tasks were able to run in a local environment
- [x] Take a screenshot of the graph view of the affected DAG in the local environment showing that all affected DAG tasks completed successfully

This takes awhile to run fully but I've confirmed with running approximately 10 files.

```
[2022-02-23 23:31:10,160] {pod_launcher.py:149} INFO - idx=8 gtfs_schedule_path='gs://gtfs-data-test/schedule/2022-02-11T00:00:00+00:00/259_0' gtfs_rt_glob_path='gtfs-data/rt/2022-02-11T00:*/259/0/*vehicle_positions*' out_dir=None result_bucket='gs://gtfs-data-test/rt-processed/validation/2022-02-11/validation_results/259/0/vehicle_positions.parquet' event='Saving aggregate counts as: gs://gtfs-data-test/rt-processed/validation/2022-02-11/validation_results/259/0/vehicle_positions.parquet'
[2022-02-23 23:31:11,853] {pod_launcher.py:149} INFO - event='finished multiprocessing; 6 successful of 10'
[2022-02-23 23:31:14,065] {pod_launcher.py:198} INFO - Event: gtfs-rt-validation.e084662ae03d48f79c63ab11c3ec5070 had an event of type Succeeded
[2022-02-23 23:31:14,065] {pod_launcher.py:311} INFO - Event with job id gtfs-rt-validation.e084662ae03d48f79c63ab11c3ec5070 Succeeded
[2022-02-23 23:31:14,154] {pod_launcher.py:198} INFO - Event: gtfs-rt-validation.e084662ae03d48f79c63ab11c3ec5070 had an event of type Succeeded
[2022-02-23 23:31:14,154] {pod_launcher.py:311} INFO - Event with job id gtfs-rt-validation.e084662ae03d48f79c63ab11c3ec5070 Succeeded
[2022-02-23 23:31:14,313] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=rt_loader, task_id=load_rt_validations, execution_date=20220211T000000, start_date=20220223T231243, end_date=20220223T233114
(.venv) ➜  airflow git:(load-rt)
```
Here's a narrow slice of the 11th
image

### CODE REVIEW

**Comment by atvaccaro** [comment]
Marking this as ready for review; I'm still working on getting it to run fully in Airflow. Bit tricky with the CSV params thing.

File: airflow/dags/rt_loader/external_validation_service_alerts.sql [hunk 1]
```diff
@@ -1,38 +1,11 @@
...
-  GTFS RT validation errors as returned by the validator. Each entry corresponds
-  to a <filename>.results.json file that is spit out by the validator.
-fields:
-  errorMessage: |
-    An object with fields for messageId, gtfsRTFeedIterationModel, validationRule,
-    errorDetails.
-  occurrenceList: |
-    An array of objects with these fields: occurenceId, messageLogModel, prefix
+dependencies:
+    - load_rt_validations
```
Comment (lauriemerrell) [inline airflow/dags/rt_loader/external_validation_service_alerts.sql:4]: Can we re-add field documentation for these?

File: airflow/dags/rt_loader/external_validation_service_alerts.sql [hunk 2]
```diff
@@ -1,38 +1,11 @@
...
-        messageLogModel STRING,
-        prefix STRING
-      >
-    >
-)
+CREATE OR REPLACE EXTERNAL TABLE gtfs_rt.validation_service_alerts
 OPTIONS (
-    uris=["gs://calitp-py-ci/gtfs-rt-validator-api/test_output_full/*gtfs_rt_service_alerts_url"],
-    format="JSON"
+    uris=["{{get_bucket()}}/rt-processed/validation/*/service_alerts.parquet"],
```
Comment (lauriemerrell) [inline airflow/dags/rt_loader/external_validation_service_alerts.sql:9]: Just confirming -- we to this point have not been loading anything, right? All the validations that are loaded are "test" data for `126` and `106` only? Do we want to run a backfill on this at some point (immediately on merge)?

File: airflow/dags/rt_loader/external_validation_service_alerts.sql (hunk 1 above)
Comment (atvaccaro) [reply airflow/dags/rt_loader/external_validation_service_alerts.sql:4]: Can do.

File: airflow/dags/rt_loader/external_validation_service_alerts.sql (hunk 2 above)
Comment (atvaccaro) [reply airflow/dags/rt_loader/external_validation_service_alerts.sql:9]: I think so, the existing job writes to a different path entirely.

A backfill will be possible, we just need to decide how far back we want to go.

**Review by lauriemerrell — APPROVED** [review]

**Pull request merged**